dts/arm/ti/am62x_m4.dtsi

```diff
@@ -10,6 +10,7 @@
 #include <zephyr/dt-bindings/pinctrl/ti-k3-pinctrl.h>
 #include <zephyr/dt-bindings/gpio/gpio.h>
 #include <zephyr/dt-bindings/i2c/i2c.h>
+#include <ti/k3-am62-main.dtsi>
```

Comment:
AFAIK the M4 has no access to main domain.

Reply:
This is extremely dangerous ... if you go into the MCU only power mode, all main domains are shut off. I don't we should do that only because it's possible.

Reply:
okay, looks like we do the same for the AM64x already ... I'm planning to rework the AM62x device trees soon to fit the latest AM64x structure. That would automatically add `main_uart` to the M4 core.